cns/requestcontroller/kubecontroller/crdtranslator.go

```diff
@@ -37,21 +39,22 @@ func CRDStatusToNCRequest(crdStatus nnc.NodeNetworkConfigStatus) (cns.CreateNetw
 		ncRequest.NetworkContainerid = nc.ID
 		ncRequest.NetworkContainerType = cns.Docker
 
-		// Convert "10.0.0.1/32" into "10.0.0.1" and prefix length
-		// Todo, this will be changed soon and only ipaddress will be passed
-		if ip, ipNet, err = net.ParseCIDR(nc.PrimaryIP); err != nil {
-			return ncRequest, err
+		if ip = net.ParseIP(nc.PrimaryIP); ip == nil {
```

Comment:
Maybe we should still try to sanitize input even in the form of "10.0.0.1/32" (only that prefix, since it's unambiguous). This might help keep things running if there were ever a regression on the other side.